apps/console/src/features/applications/components/forms/general-details-form.tsx

```diff
@@ -162,9 +171,10 @@ export const GeneralDetailsForm: FunctionComponent<GeneralDetailsFormPopsInterfa
     const { getLink } = useDocumentation();
 
     const UIConfig: UIConfigInterface = useSelector((state: AppState) => state?.config?.ui);
+    const featureConfig: FeatureConfigInterface = useSelector((state: AppState) => state?.config?.ui?.features);
 
     const [ isDiscoverable, setDiscoverability ] = useState<boolean>(discoverability);
-
+    const [ saas, setSaas ] = useState<boolean>(isSaasApp);
```

Comment:
shall we rename this so that it wouldn't get confused with the field name?

Reply:
Fixed with ff72028